### ISSUE TITLE
Possible Perl installation issue

### ISSUE THREAD

**Issue by erasche** [galaxyproject/tools-iuc#165]
```
Installing /home/galaxy/galaxy/tool_dependencies/perl/5.18.1/iuc/package_perl_5_18/f6efd799dc70/local-lib/man/man3/POD2::PT_BR::local::lib.3                                                                 
Appending installation info to /home/galaxy/galaxy/tool_dependencies/perl/5.18.1/iuc/package_perl_5_18/f6efd799dc70/local-lib/lib/perl5/x86_64-linux/perllocal.pod                                           
Can't locate local/lib.pm in @INC (you may need to install the local::lib module) (@INC contains: /home/galaxy/galaxy/tool_dependencies/perl/5.18.1/iuc/package_perl_5_18/f6efd799dc70/local-lib/perl5 /home#
BEGIN failed--compilation aborted.                                                                                                                                                                           
Can't write to cpanm home '/.cpanm': You should fix it with chown/chmod first.                                                                                                                               
```

During testing installation from TTS.

**Comment by erasche** [comment]
```
# ls /home/galaxy/galaxy/tool_dependencies/perl/5.18.1/iuc/package_perl_5_18/f6efd799dc70/local-lib/perl5
ls: cannot access /home/galaxy/galaxy/tool_dependencies/perl/5.18.1/iuc/package_perl_5_18/f6efd799dc70/local-lib/perl5: No such file or directory


# ls /home/galaxy/galaxy/tool_dependencies/perl/5.18.1/iuc/package_perl_5_18/f6efd799dc70/local-lib/lib/perl5/
lib  local  POD2  x86_64-linux
```

**Comment by erasche** [comment]
CC @bgruening

**Comment by mvdbeek** [comment]
I guess this could also be fixed with PR #253
